src/api/subscriptions.js

```diff
@@ -273,6 +273,9 @@ export class Subscriptions {
 
   /** @return {!Promise<LoggerApi>} */
   getLogger() {}
+
+  /** @return {!ClientEventManager} } */
+  getEventManager() {}
```

Comment:
Should we be concerned about them trying to become filterers and squashing out logging?

Reply:
We had initially created logger as their version of event manager (it was only working in propensity events granted).  We could extend listeners through that if we are concerned about filterers.  Not sure we care.  Other than that this seems fine, as long as they have access to AnalyticsEvent already also

Reply:
It's their data and given the limitless ways publisher can screw up the data filtering is the least of our worries.